sql/core/src/test/scala/org/apache/spark/sql/connector/WriteDistributionAndOrderingSuite.scala

```diff
@@ -77,36 +101,47 @@ class WriteDistributionAndOrderingSuite extends DistributionAndOrderingSuiteBase
   }
 
   test("ordered distribution and sort with same exprs with numPartitions: append") {
-    checkOrderedDistributionAndSortWithSameExprs("append", Some(10))
+    checkOrderedDistributionAndSortWithSameExprs("append", Some(10), true, false, false)
   }
 
   test("ordered distribution and sort with same exprs with numPartitions: overwrite") {
-    checkOrderedDistributionAndSortWithSameExprs("overwrite", Some(10))
+    checkOrderedDistributionAndSortWithSameExprs("overwrite", Some(10), true, false, false)
   }
 
   test("ordered distribution and sort with same exprs with numPartitions: overwriteDynamic") {
-    checkOrderedDistributionAndSortWithSameExprs("overwriteDynamic", Some(10))
+    checkOrderedDistributionAndSortWithSameExprs("overwriteDynamic", Some(10), true, false, false)
   }
 
   test("ordered distribution and sort with same exprs with numPartitions: micro-batch append") {
-    checkOrderedDistributionAndSortWithSameExprs(microBatchPrefix + "append", Some(10))
+    checkOrderedDistributionAndSortWithSameExprs(microBatchPrefix + "append", Some(10),
+      true, false, false)
   }
 
   test("ordered distribution and sort with same exprs with numPartitions: micro-batch update") {
-    checkOrderedDistributionAndSortWithSameExprs(microBatchPrefix + "update", Some(10))
+    checkOrderedDistributionAndSortWithSameExprs(microBatchPrefix + "update", Some(10),
+      true, false, false)
   }
 
   test("ordered distribution and sort with same exprs with numPartitions: micro-batch complete") {
-    checkOrderedDistributionAndSortWithSameExprs(microBatchPrefix + "complete", Some(10))
+    checkOrderedDistributionAndSortWithSameExprs(microBatchPrefix + "complete", Some(10),
+      true, false, false)
   }
 
-  private def checkOrderedDistributionAndSortWithSameExprs(command: String): Unit = {
-    checkOrderedDistributionAndSortWithSameExprs(command, None)
+  private def checkOrderedDistributionAndSortWithSameExprs(
```

Comment:
seems we can have only one `checkOrderedDistributionAndSortWithSameExprs` and provide default values for all the parameters except for `command: String`

Reply:
Changed. Thanks!